src/character/mod.rs

```diff
@@ -152,6 +183,37 @@ impl Character {
         self
     }
 
+    /// Applies ability bonuses to the character's statistics.
+    fn apply_ability_bonuses(&mut self) {
+        self.apply_constitution_bonus();
+    }
+
+    fn apply_constitution_bonus(&mut self) {
```

Comment:
I don't know if it's a good place, but should we somewhere somehow handle the case when constitution drops to 0? I think somewhere it meant instant death, same as level drain. But perhaps it should be handled on the spell / condition level.

Reply:
I need to look up if CON can still be 0, I think now the absolute minimum for it is 1. 5th edition did simply some stuff.

Anyway we can handle it when the character takes damage,